manifest-editor/components/molecules/AddManifestModal.tsx

```diff
@@ -51,6 +55,11 @@ export const AddManifestModal: React.FC<{
     if (inputed.type === "Manifest") onChange(inputValue);
   };
 
+  const handleImage = (e: any) => {
+    setHeight(e.height);
+    setWidth(e.width);
```

Comment:
We could use [naturalHeight](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/naturalHeight) and width properties here.
```
e.naturalHeight
e.naturalWidth
```
They should ensure the height/width match the image loaded (and not the image tag) just in case.